src/Microsoft.AspNet.Mvc.Core/ActionResults/FileContentResult.cs

```diff
@@ -19,13 +20,14 @@ public class FileContentResult : FileResult
 
         /// <summary>
         /// Creates a new <see cref="FileContentResult"/> instance with
-        /// the provided <paramref name="fileContents"/>.
+        /// the provided <paramref name="fileContents"/> and the
+        /// provided <paramref name="contentType"/>.
         /// </summary>
         /// <param name="fileContents">The bytes that represent the file contents.</param>
-        public FileContentResult([NotNull] byte[] fileContents)
-            : base(contentType: null)
+        /// <param name="contentType">The Content-Type header of the response.</param>
+        public FileContentResult([NotNull] byte[] fileContents, string contentType)
```

Comment:
Please review all of these and add `[NotNull]`. The idea is to throw as close as possible to where the mistake occurred.